pkg/k8shandler/util.go

```diff
@@ -126,9 +128,93 @@ func isValidConf(dpl *api.Elasticsearch) error {
 		}
 	}
 
+	if ok, msg := hasValidUUIDs(dpl); !ok {
+		if err := updateInvalidUUIDChangeCondition(dpl, v1.ConditionTrue, msg); err != nil {
+			return err
+		}
+		return fmt.Errorf("Unsupported change to UUIDs made: %v", msg)
```

Comment:
I think this is a good reason to prefer annotation over field.  Annotations are targeted for 'tools' or places where the platform can keep information or have hints where field almost invites a user to change something